src/Targeting/ActionHandler/TrackSegment.php

```diff
@@ -42,8 +41,7 @@ class TrackSegment implements ActionHandlerInterface, DataProviderDependentInter
 
     public function __construct(
         SegmentManagerInterface $segmentManager,
-        SegmentTracker $segmentTracker,
-        EventDispatcherInterface $eventDispatcher
```

Comment:
I'd go for deprecation, instead of direct removal for BC reasons.

Reply:
Revert constructor changes and add TODO https://github.com/pimcore/customer-data-framework/pull/248/commits/83c5ac253ca423daaff63d02922b19d97b61fd01